### PR TITLE
Extend logging for scalability tests debugging

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #32257

Risk: very low

Importancy: not neeted (though it might be easier to cherrypick #32275 with it)

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32269)

### CODE REVIEW

File: pkg/storage/cacher.go [hunk 1]
```diff
@@ -643,6 +650,7 @@ func (c *cacheWatcher) add(event watchCacheEvent) {
 		c.forget(false)
 		c.stop()
 	}
+	glog.V(2).Infof("cacheWatcher blocked processing for %v", time.Since(startTime))
```
Comment (gmarek) [inline pkg/storage/cacher.go]: cacheWatcher add function blocked...

Comment (wojtek-t) [inline pkg/storage/cacher.go]: done

**Comment by wojtek-t** [comment]
@gmarek PTAL

**Comment by pwittrock** [comment]
@wojtek-t @gmarek 

1.  Please provide a risk assessment in the description

2. How critical is this?  blocking / important / best effort?  We have dozens of cherrypick requests and each one creates a lot of overhead because large batches introduce a high-risk of destabilizing the tests in a way that is hard to track down.   We have to break them up into smaller batches and monitor the tests to catch issues early on and track them back to their root cause.

**Comment by wojtek-t** [comment]
@pwittrock - this isn't important, though risk for it is super low, and it might be easier to cherrypick #32275 with this one.

**Pull request merged**